src/components/layout/AddMovie.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+export const AddMovie = () => {
+  return (
+    <div className='addMovie_container '>
+      <div className='addMovie_tag '>
+         movies<label style={{fontWeight:'bold'}}>DB</label>
+      </div>
+      <div className='addMovie_btn border'>+ADD MOVIE</div>
```

Comment:
This is also semantic but kind of important: if you need a button use `button` or at least `a`.